lib/index.js

```diff
@@ -92,12 +94,54 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         });
     }));
 
+  app.get(
+    adminPath,
+    ensureAuthenticated,
+    intParser(),
+    validate({query: 'account.pagination'}),
+    asyncHandler(async (req, res) => {
+      const {email, cursor = null, limit = 10} = req.query;
+      const {actor} = req.user || {};
+      const query = {"account.email": email};
+      if(cursor) {
+        query["account.created"] = {$gte: cursor};
+      }
+      const options = {limit, sort: {"account.created": -1}};
+      const records = await brAccount
+        .getAll({actor, query, options});
+      res.status(200).json(records);
+    }));
+
   app.patch(
     accountPath,
     ensureAuthenticated,
-    asyncHandler(async (req, res, next) => {
-      // TODO: call `brAccount.update`
-      next();
+    intParser(),
+    validate('account.update'),
+    asyncHandler(async (req, res) => {
+      /**
```

Comment:
not a named function, and I'm not sure about documenting express things like this.  I think @davidlehn pointed you at how we document HTTP APIs.

Reply:
we do have to figure out how to document these functions.
we can't just expect people to crawl through package.json back a project or two to figure out we are using a json-patch library etc. there are js-doc comments in other files:

https://github.com/digitalbazaar/bedrock-account/blob/master/lib/index.js

removing the comments will make it significantly harder.
we have to documentation and the RAML stuff is not quite there.
if we allow js-doc in bedrock-account why not allow it here?

Reply:
@dlongley  @davidlehn @dmitrizagidulin  @msporny  I think we really do need to comment this file. using raml for the api end points is ok, but we already use js-doc in bedrock-account
plus the comments on this handler are really important because they link to the docs for:
fast-patch-json and also explain the structure of the body object.  js-doc can be stripped out of production code easily while RAML might not be as easy. The major reason for this comment is that I had to backtrack to bedrock-account to what a patch is etc. this just makes it plain and should make dev time quicker in the future.

Reply:
I might add try looking at this file with no comments and figure out what each routes does. we even use variables obscuring the path names etc.

Reply:
also of note RAML actually documents http stuff and not javascript stuff so it for instance documents route, route name, route method, route return codes and headers etc. it's actually different than the documentations needed for the handlers.